client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/logging/AbstractLogCollector.java

```diff
@@ -231,4 +227,15 @@ public void run() {
             }, delay, TimeUnit.SECONDS);
         }
     }
+
+    public final void scheduleAtFixedRateLogUpload() {
```

Comment:
This is wrong implementation. We do not need to schedule some periodic checks in all cases. We need to schedule only one check when log is actually sent.